api/proto/teleport/access_graph/v1/secrets_service.proto

```diff
@@ -48,20 +65,16 @@ message ReportAuthorizedKeysRequest {
 // RPC method.
 message ReportAuthorizedKeysResponse {}
 
-// Request for ReportSecrets.
-//
-// Authentication ceremony flow:
-// -> AuthenticateDeviceInit (client)
-// <- AuthenticateDeviceChallenge (server)
-// -> AuthenticateDeviceChallengeResponse (client)
-// <- DeviceAuthentication (server)
-// -> ReportPrivateKeys (client) [1 or more]
-// -> CloseStream (client)
+// ReportSecretsRequest is used by trusted devices to report secrets found on the host
+// that could be used to bypass Teleport.
 message ReportSecretsRequest {
   oneof payload {
-    AuthenticateDeviceInit init = 1;
-    teleport.devicetrust.v1.AuthenticateDeviceChallengeResponse challenge_response = 2;
-    teleport.devicetrust.v1.TPMAuthenticateDeviceChallengeResponse tpm_challenge_response = 3;
+    // The device should initiate the device assertion ceremony by sending the
+    // AssertDeviceRequest.
+    // The server will respond with a challenge that the device should respond to.
+    // Once the device is asserted, the client can send the secrets using the [private_keys] field.
```

Comment:
Same as above, just tell folks to lookup AssertDeviceRequest (or simply remove the docs).